index.js

```diff
@@ -10,7 +10,7 @@ import {visit} from 'unist-util-visit'
 const relative = /^\.{1,2}\//
 
 /**
- * Plugin to embed local images as data URIs, inlining files as base64-encoded
+ * Plugin to embed local images as data URIs, inlining files as encoded (base64)
```

Comment:
parenthesis is a nice add, the original order seemed to flow a bit better, thoughts?
```suggestion
 * Plugin to embed local images as data URIs, inlining files as (base64) encoded
```

Reply:
I’m personally not a fan of the added parentheres. I’d leave them out. I do like that the hyphen has been removed.

Reply:
Maybe the whole second phrase can be removed, as it’s implied by the earlier phrase already?